### ISSUE TITLE
`orb` variable name in examples

### ISSUE THREAD

**Issue by zaccolley** [orbotix/sphero.js#44]
With products that aren't orbs (like the Ollie) would it be better to have examples that have a more neutral name than `orb`?

Could do `device`? Not sure about it, sounds a bit crap. Thoughts?

**Comment by deadprogram** [comment]
How about `robot`?

**Comment by zaccolley** [comment]
Yeah that's cool, I've noticed orb is used throughout the codebase so not as simple change as I thought
